includes/class-wcmp-rest.php

```diff
@@ -131,14 +131,18 @@ public function request($url, $method = "GET", $headers = array(), $post, $body
             @fclose($f);
         }
 
+        if (! empty($response->errors) && array_key_exists('http_request_failed', $response->errors)){
+            echo 'There is a technical problem with connecting to MyParcel. We are working hard to resolve the problem.';
+            return array("code" => 404, "body" => $response->errors);
```

Comment:
Wat zit er in ->errors (een array)?

Reply:
Een string met de melding erin dat er geen connectie gemaakt kan worden.